package.json

```diff
@@ -24,6 +24,7 @@
     "@mui/material": "^5.0.6",
     "@sindresorhus/slugify": "<2",
     "@types/js-levenshtein": "^1.1.0",
+    "clsx": "^1.1.1",
```

Comment:
This is just for our convenience. Material-UI already has clsx as a dependency, so this doesn't change our bundle size at all.

But this means we can just use
```ts
import cx from "clsx";

// and later on...
<div className={cx("a", someCondition && "b")} />
```
Instead of this jank we had to use before:
```ts
import { ClassNames } from "@emotion/react";

// ...
<ClassNames>
  {({ cx }) => (
    <div className={cx("a", someCondition && "b")} />
  )}
</ClassNames>
```